fili-sql/src/test/java/com/yahoo/bard/webservice/database/Database.java

```diff
@@ -0,0 +1,182 @@
+    public static final String IS_ROBOT = "isRobot";
+    public static final String PAGE = "page";
+    public static final String USER = "user";
+    public static final String COMMENT = "comment";
+    public static final String ADDED = "added";
+    public static final String DELETED = "deleted";
+    public static final String DELTA = "delta";
+    public static final String ID = "ID";
+    public static final String METRO_CODE = "metroCode";
+
+    /**
+     * Gets an in memory database with the {@link WikitickerEntry} from the example data.
+     *
+     * @return the connection to the database.
+     *
+     * @throws SQLException if can't create database correctly.
+     * @throws IOException if can't read example data file.
+     */
+    public static Connection initializeDatabase() throws SQLException, IOException {
+        if (connection == null) {
+            connection = DriverManager.getConnection(DATABASE_URL);
+        } else {
+            return connection;
+        }
+
+        List<WikitickerEntry> entries = readJsonFile();
+
+        Statement s = connection.createStatement();
+        s.execute("CREATE TABLE \"" + WIKITICKER + "\" (\"" + ID + "\" INT PRIMARY KEY, \"" +
+                COMMENT + "\" VARCHAR(256), \"" +
+                COUNTRY_ISO_CODE + "\" VARCHAR(256), \"" +
+                ADDED + "\" INTEGER,\"" +
+                TIME + "\" TIMESTAMP, \"" +
+                IS_NEW + "\" BOOLEAN, \"" +
+                IS_ROBOT + "\" BOOLEAN, \"" +
+                DELETED + "\" INTEGER, \"" +
+                METRO_CODE + "\" VARCHAR(256), \"" +
+                IS_UNPATROLLED + "\" BOOLEAN, \"" +
+                NAMESPACE + "\" VARCHAR(256), \"" +
+                PAGE + "\" VARCHAR(256), \"" +
+                COUNTRY_NAME + "\" VARCHAR(256), \"" +
+                CITY_NAME + "\" VARCHAR(256), \"" +
+                IS_MINOR + "\" BOOLEAN, \"" +
+                USER + "\" VARCHAR(256), \"" +
+                DELTA + "\" INTEGER, \"" +
+                IS_ANONYMOUS + "\" BOOLEAN, \"" +
+                REGION_ISO_CODE + "\" VARCHAR(256), \"" +
+                CHANNEL + "\" VARCHAR(256), \"" +
+                REGION_NAME + "\" VARCHAR(256), " +
+                ")"
+        );
+
+        String sqlInsert = "INSERT INTO \"" + WIKITICKER + "\" (\"" + ID + "\", \"" + COMMENT + "\", \"" +
+                COUNTRY_ISO_CODE + "\", \""
+                + ADDED + "\", \"" + TIME + "\", \"" + IS_NEW + "\", \"" + IS_ROBOT + "\", \"" + DELETED + "\", \"" +
+                METRO_CODE + "\", \"" +
+                IS_UNPATROLLED + "\", \"" + NAMESPACE + "\", \"" + PAGE + "\", \"" + COUNTRY_NAME + "\", \"" +
+                CITY_NAME + "\", \"" +
+                IS_MINOR + "\", \"" + USER + "\", \"" + DELTA + "\", \"" + IS_ANONYMOUS + "\", \"" + REGION_ISO_CODE
+                + "\", \"" + CHANNEL
+                + "\", \"" + REGION_NAME + "\") VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
If we could have some comments with a more interpolated view of these SQL statements (i.e. INSERT INTO $WIKITICKER...), that would be wonderful.

This is making my eyes bleed.